build.gradle.kts

```
@@ -0,0 +1,46 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+
+plugins {
+	kotlin("jvm") version "1.9.24"
+	kotlin("plugin.spring") version "1.9.24"
+	id("org.springframework.boot") version "3.3.2"
+	id("io.spring.dependency-management") version "1.1.6"
+	kotlin("plugin.jpa") version "1.9.24"
```

```suggestion
    kotlin("plugin.jpa") version "1.9.24"
```
Pls, fix the whole file